lib/src/pubspec.dart

```diff
@@ -121,6 +128,16 @@ class Pubspec {
         'iosPrefix',
         'pluginClass',
       }.any((_inner.flutter['plugin'] as Map<String, Object>).containsKey);
+
+  /// 'true' if any dart2 versions of the sdk are supported.
+  bool get supportsDart2 {
```

Comment:
I think this could reuse `SdkConstraintStatus`.

Reply:
Ah - true, that was already there!